src/api/app/jobs/cleanup_events.rb

```diff
@@ -1,10 +1,7 @@
 class CleanupEvents < ApplicationJob
   def perform
     Event::Base.transaction do
-      Event::Base.where(project_logged: true, mails_sent: true, undone_jobs: 0).lock(true).delete_all
+      Event::Base.where(mails_sent: true, undone_jobs: 0).lock(true).delete_all
     end
-
-    event_types = Event::PROJECT_CLASSES | Event::PACKAGE_CLASSES
-    Event::Base.where(project_logged: false, mails_sent: true, undone_jobs: 0).where.not(eventtype: event_types).delete_all
```

Comment:
I am not pretty sure why this line is not in a transaction and the one up is 🤔

Reply:
There is no real need to use a transaction.. I added this line as a quick fix until the project_logged column could be deleted, so you have done the right thing by deleting it.

Reply:
👍 I delete the transaction then